src/Reflection/Adapter/ReflectionType.php

```diff
@@ -7,20 +7,18 @@
 use ReflectionType as CoreReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionIntersectionType as BetterReflectionIntersectionType;
 use Roave\BetterReflection\Reflection\ReflectionNamedType as BetterReflectionNamedType;
+use Roave\BetterReflection\Reflection\ReflectionType as BetterReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionUnionType as BetterReflectionUnionType;
 
 use function array_filter;
 use function array_values;
+use function assert;
 use function count;
 
 abstract class ReflectionType extends CoreReflectionType
 {
-    public static function fromTypeOrNull(BetterReflectionNamedType|BetterReflectionUnionType|BetterReflectionIntersectionType|null $betterReflectionType): ReflectionUnionType|ReflectionNamedType|ReflectionIntersectionType|null
+    public static function fromType(BetterReflectionType $betterReflectionType): ReflectionUnionType|ReflectionNamedType|ReflectionIntersectionType
```

Comment:
The union type would still be better here, compared to `BetterReflectionType`, which could be extended by third parties :thinking:

Reply:
Fixed.